dashboard/app/models/activity.rb

```diff
@@ -72,4 +72,62 @@ def Activity.recent(limit)
     # yeah, this is a lot like .last -- but I want a dataset not an array
     Activity.order('id desc').limit(limit)
   end
+
+  # Creates a new Activity which will be written eventually to the database. (Note that the id is
+  # nil because it may not have been written yet.) An exception will be thrown if the object does
+  # not pass validation. The object is only written asynchronously if the gatekeeper allows it for
+  # this hostname.
+  def Activity.create_async!(attributes)
+    if Gatekeeper.allows('async_activity_writes', where: {hostname: Socket.gethostname})
```

Comment:
That's too bad.  I think it's OK though;  I just want to canary it on a few servers initially (where I can look up the ip- hostname) and then switch it entirely on or entirely off.